pkg/ring/kv/memberlist/memberlist_client.go

```diff
@@ -122,7 +123,9 @@ type valueDesc struct {
 
 var (
 	// if merge fails because of CAS version mismatch, this error is returned. CAS operation reacts on it
```

Comment:
You can probably trim this commend, these error messages are self documenting for all intents and purposes